kucher/view/main_window/register_view_widget/import_export_dialog.py

```diff
@@ -0,0 +1,310 @@
+#
+# Copyright (C) 2018 Zubax Robotics OU
+#
+# This file is part of Kucher.
+# Kucher is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+# Kucher is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with Kucher.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Author: Pavel Kirienko <pavel.kirienko@zubax.com>
+#
+
+import os
+import yaml
+import asyncio
+import math
+import itertools
+
+from logging import getLogger
+from PyQt5.QtWidgets import QWidget, QFileDialog, QTableWidget, QTableWidgetItem, QDialog, QLabel, \
+    QPushButton, QHeaderView, QMessageBox
+from popcop.standard.register import ValueType
+
+from kucher.view.device_model_representation import Register
+from kucher.view.utils import show_error, lay_out_horizontally, lay_out_vertically, get_monospace_font
+
+_logger = getLogger(__name__)
+
+
+def export_registers(parent, registers):
+    """
+    Store all mutable and persistent registers in a .yml file
+    """
+    dialog_box = QWidget()
+    dialog_box.setGeometry(10, 10, 1000, 700)
+
+    file_name, _ = QFileDialog.getSaveFileName(dialog_box, 'Export configuration file', 'config.yml',
+                                               'YAML Files (*.yml  *.yaml);;All Files (*)')
+    if not file_name:
+        return
+    try:
+        _file = open(file_name, 'w+')
+
+    except Exception as ex:
+        show_error('Export failed',
+                   f'Cannot export {file_name}',
+                   f'Error: {ex}',
+                   parent)
+        _logger.exception(f'File {file_name} could not be exported: {ex!r}')
```

Comment:
```suggestion
        _logger.exception(f'File {file_name} could not be open: {ex!r}')
```

Makes the error message more specific.